test/jdk/tools/launcher/Arrrghs.java

```diff
@@ -601,6 +601,38 @@ void testMainMethod() throws FileNotFoundException {
         tr.checkPositive();
         if (!tr.testStatus)
             System.out.println(tr);
+
+        //private method with parameter, usable method without parameter:
+        createJar(new File("some.jar"), new File("Foo"),
+            "private static void main(String[] args){}",
```

Comment:
Perhaps we should even include a instance method `private void main(String[] args) {}` to cover that case?

Reply:
Done:
https://github.com/openjdk/jdk/pull/25256/commits/a351686d1aeb7a82f52141558ca7d2590cb7689f